src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurSaeCreateRequestedCashAdvanceFileValidationServiceImpl.java

```diff
@@ -93,22 +92,22 @@ private boolean requestDetailFileLineIsCashAdvance(ConcurStandardAccountingExtra
     }
 
     /**
-     * Requested cash advances issued/approved by Concur Cash Advance Administrator will have a cash advance 
-     * transaction type indicating that approval/issuance.  These are the lines that need to be processed as requested
-     * cash advances when additional validation also passes.
+     * Requested cash advances are either valid if they are either approved by administrator (1) or applied to trip reimbursement (2)
      *
      * @param detailFileLine
      * @return boolean
      */
-    private boolean requestedCashAdvanceApprovedByConcurAdministrator(ConcurStandardAccountingExtractDetailLine detailFileLine) {
-        if (getConcurStandardAccountingExtractCashAdvanceService().isPreTripCashAdvanceIssuedByCashAdmin(detailFileLine)) {
-            detailFileLine.getValidationResult().setCashAdvanceAdministratorApprovedCashAdvance(true);
+    private boolean requestedCashAdvanceApprovedOrApplied(ConcurStandardAccountingExtractDetailLine detailFileLine) {
+        boolean validCashAdvance = getConcurStandardAccountingExtractCashAdvanceService().isPreTripCashAdvanceIssuedByCashAdmin(detailFileLine) ||
+                getConcurStandardAccountingExtractCashAdvanceService().isPreTripCashAdvanceIssuedByCashAdmin(detailFileLine);
```

Comment:
This section is calling the `isPreTripCashAdvanceIssuedByCashAdmin()` method twice with the same arguments. Did you mean to replace one of those calls with `isCashAdvanceToBeAppliedToReimbursement()` instead?